lib/sales_engine.rb

```diff
@@ -0,0 +1,24 @@
+# require 'csv'
+require_relative './customer_repository'
+require_relative './invoice_repository'
+require_relative './invoice_item_repository'
+require_relative './item_repository'
+require_relative './merchant_repository'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.